### PR TITLE
feature/books-list

### PR DESCRIPTION
**Pull request by GCerbelli**
## Summary

- Add book list component

## Trello Card

[Nº2.3 Lista de libros](https://trello.com/c/tPV9J6yZ/9-n23-lista-de-libros)

## Screenshots
image

### CODE REVIEW

File: wbooks/src/app/components/Books/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import BookCard from '../BookCard';
+import { BOOKS_DATA } from '../../../constants/mocks';
+import { Book } from '../../interfaces/books';
+import { styles } from '../Books/styles';
+
+const renderItem = ({ item }: { item: Book }) => (
+  <BookCard title={item.title} author={item.author} img={item.img} />
+);
+const renderSeparator = () => <View style={styles.bookCardSeparator} />;
```
Comment (guidoprinc) [inline wbooks/src/app/components/Books/index.tsx:12]: this renders must be inside the component function so they only persist there

Comment (guidoprinc) [reply wbooks/src/app/components/Books/index.tsx:12]: also in the future you will need more values that will be obtained only in the component

File: wbooks/src/app/components/Books/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View, FlatList } from 'react-native';
+
+import BookCard from '../BookCard';
+import { BOOKS_DATA } from '../../../constants/mocks';
+import { Book } from '../../interfaces/books';
+import { styles } from '../Books/styles';
+
+const renderItem = ({ item }: { item: Book }) => (
```
Comment (guidoprinc) [inline wbooks/src/app/components/Books/index.tsx:9]: move this into BooksList

File: wbooks/src/constants/colors.ts [hunk 3]
```diff
@@ -1,2 +1,3 @@
 export const white = '#FFFFFF';
 export const dark = '#444444';
+export const lightBlueBackgroud = '#EAF6FB';
```
Comment (marianozicavo) [inline wbooks/src/constants/colors.ts:3]:
```suggestion
export const lightBlue = '#EAF6FB';
```

File: wbooks/src/app/components/Books/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { View, FlatList } from 'react-native';
+
+import BookCard from '../BookCard';
+import { BOOKS_DATA } from '../../../constants/mocks';
+import { Book } from '../../interfaces/books';
+import { styles } from '../Books/styles';
+
+const BooksList = () => {
+  const renderItem = ({ item }: { item: Book }) => (
+    <BookCard title={item.title} author={item.author} img={item.img} />
```
Comment (marianozicavo) [inline wbooks/src/app/components/Books/index.tsx:11]: Did it show you some warning about missing keys? Any ways you should still add it:
```suggestion
    <BookCard key={item.id} title={item.title} author={item.author} img={item.img} />
```

File: wbooks/App.tsx [hunk 5]
```diff
@@ -10,12 +10,12 @@
 
 import React from 'react';
 
-import BookCard from './src/app/components/BookCard';
+import BooksList from './src/app/components/Books';
```
Comment (guidoprinc) [inline wbooks/App.tsx:13]: the folder should be called the same as the component

File: wbooks/src/app/components/Books/index.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, FlatList } from 'react-native';
+
+import BookCard from '../BookCard';
+import { BOOKS_DATA } from '../../../constants/mocks';
+import { Book } from '../../interfaces/books';
+import { styles } from '../Books/styles';
```
Comment (guidoprinc) [inline wbooks/src/app/components/Books/index.tsx:7]: styles should be the default export

File: wbooks/src/app/components/BooksList/index.tsx [hunk 7]
```diff
@@ -4,11 +4,11 @@ import { View, FlatList } from 'react-native';
 import BookCard from '../BookCard';
 import { BOOKS_DATA } from '../../../constants/mocks';
 import { Book } from '../../interfaces/books';
-import { styles } from '../Books/styles';
+import styles from '../BooksList/styles';
```
Comment (marianozicavo) [inline wbooks/src/app/components/BooksList/index.tsx:7]:
```suggestion
import styles from './styles';
```

**Pull request merged**